topics/transcriptomics/tutorials/mirna-target-finder/tutorial.md

```diff
@@ -0,0 +1,726 @@
+> 4. {% tool [Merge Collections](__MERGE_COLLECTION__) %} with the following parameters:
+>    - In *"Input collections"*:
+>        - *"1.Input Collections"*: `FastQC unprocessed control mRNA: RawData`
+>        - *"2.Input Collections"*: `FastQC unprocessed BR treated mRNA: RawData`
+> 5. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.8+galaxy1) %} with the following parameters:
+>    - In *"Results"*:
+>      - *"Which tool was used generate logs?"*: `FastQC`
+>      - {% icon param-collection %} *"Dataset collection"*: select the output generated in the previous step.
+>   - In *"Report title"*: `mRNA initial quality check`
+> 4. Click on the {% icon galaxy-eye %} (eye) icon and inspect the generated HTML file
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> Is there any parameters that indicate the need to process the samples in order to improve its quality?
+>
+> > ### {% icon solution %} Solution
+> >
+> > All parameters are within accepted limits. However, the adapter content shows the presence of universal adapters in our readings, which can be removed to avoid possible interferences at later stages.
+> >
+> >    ![Experimental design](../../images/miRNA_adapter_01.png "Experimental design")
+> >
+> {: .solution}
+{: .question}
+
+> ### {% icon comment %} Comments
+> Although our samples have adapters, we are not going to trim them in this case. We will explain the reason in the next section.
+{: .comment}
+
+## Quantification of gene expression: **Salmon**
+
+After performing the quality assessment of the reads, we can move on to quantifying the gene expression. The aim of this step is to identify which transcript each read comes from and the total number of reads associated with each transcript. In this tutorial we will use the [Salmon](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5600148/) tool for the quantification of mRNA transcripts.
+
+![Experimental design](../../images/salmon_logo.png)
+
+One of the characteristics of __Salmon__ is that it doesn't require to perform a base-to-base alignment, which is the time-consuming step of tools such as STAR and HISAT2. Salmon relies in the quasi-mapping concept, a new mapping technique that allows the rapid and accurate mapping of RNA-seq reads to a target transcriptome. Rather than a standard alignment, quasi-mapping seeks to find the best mappings for each read, and does so by finding minimal collections of dynamically sized, right-maximal, matching contexts between target and query positions ({% cite Srivastava2016 %})
+
+The quasi-mapping approach utilized by __Salmon__ requires a reference index to determine the position and orientation information for where the fragments best map prior to quantification. It allows to provide the transcriptome in a format that is easily and rapidly searchable. Therefore, it will allow us to quickly find the positions in the transcriptome where each of the reads originated.   
```

Comment:
this hard to understand for me, especially the 'for where'

Reply:
This paragraph has been modified by Pavan, I think that now it is much more clear :)